### ISSUE TITLE
Running mpsyt in background - TypeError

### ISSUE THREAD

**Issue by n1tr0-5urf3r** [mps-youtube/mps-youtube#318]
I'm trying to run mps-youtube in background on my raspberry pi using wheezy and mplayer.
When running the following command (as mentioned in  #108) I get an error.
Command:
```
 mpsyt playurl https://www.youtube.com/watch?v=BHRyMcH6WMM > /dev/null < /dev/null &
```

Error:
```
File "/usr/local/bin/mpsyt", line 9, in <module>
    load_entry_point('mps-youtube==0.2.5', 'console_scripts', 'mpsyt')()
  File "/usr/local/lib/python3.2/dist-packages/mps_youtube/main.py", line 3916, in main
    set_window_title("mpsyt")
  File "/usr/local/lib/python3.2/dist-packages/mps_youtube/util.py", line 114, in set_window_title
    sys.stdout.write(xenc('\x1b]2;' + title + '\x07'))
TypeError: must be str, not bytes
```

**Comment by ids1024** [comment]
This should be fixed by e9516ef5ed35a8dc462f0373cec6ff067a60b24d in the develop branch.

**Comment by n1tr0-5urf3r** [comment]
Worked for me! Thank you
